chromadb/api/segment.py

```diff
@@ -230,10 +232,34 @@ def get_collection(
                 name=existing[0]["name"],
                 metadata=existing[0]["metadata"],  # type: ignore
                 embedding_function=embedding_function,
+                tenant=tenant,
+                database=database,
             )
         else:
             raise ValueError(f"Collection {name} does not exist.")
 
+    @trace_method("SegmentAPI.get_collection_by_id", OpenTelemetryGranularity.OPERATION)
+    @override
+    def get_collection_by_id(
```

Comment:
why not just overload get_collection() with an id parameter?

Reply:
Makes sense. Adding this now.